src/promptflow/promptflow/_sdk/_service/apis/span.py

```diff
@@ -57,6 +65,21 @@ def from_request() -> "ListSpanParser":
         SpanResourceFieldName.SCHEMA_URL: fields.String,
     },
 )
+event_model = api.model(
+    "Event",
```

Comment:
may add some example value on event links in PR description

Reply:
or add some test with such field

Reply:
updated the pr desc